src/runtime_src/core/common/api/xrt_bo.cpp

```diff
@@ -1171,7 +1173,8 @@ fill_copy_pkt(const xrt::bo& dst, const xrt::bo& src, size_t sz,
 #ifndef _WIN32
```

Comment:
Sorry, didn't mean to remove Stephen's comment.   He said this should be refactored to call common code as it is not supported on Windows.  I wanted to reply, I didn't understand the comment?